Sources/Preferences/PreferencePane.swift

```diff
@@ -1,6 +1,6 @@
 import Cocoa
 
-public struct PreferencePaneIdentifier: Equatable, RawRepresentable {
+public struct PreferencePaneIdentifier: Equatable, RawRepresentable, Hashable {
```

Comment:
`Hashable` inherits from `Equatable`, so you don't need both.

```suggestion
public struct PreferencePaneIdentifier: Hashable, RawRepresentable {
```